File: assignment3/ElectricCar.py
from Car import Car

class ElectricCar(Car):
    def __init__(self, makeAndModel="None assigned", numberOfDoors=4, maximumNumberOfPassengers=5, batterySize=-1) -> None:
        super().__init__(makeAndModel, numberOfDoors, maximumNumberOfPassengers)
        self.batterySize = batterySize

    def __str__(self):
        defaultStr = super().__str__()
        return f'{defaultStr}\nBattery size: {self.batterySize}'

    def getBatterySize(self):
        return self.batterySize
    
    def setBatterySize(self, newBatterySize):
        self.batterySize = newBatterySize

File: Assignment4/ElectricCar.java
package Assignment4;
public class ElectricCar extends Car {
    double batterySize;

    public ElectricCar(String makeModel, int numPassengers, int numDoors, double battSize) {
        super(makeModel, numPassengers, numDoors);
        batterySize = battSize;
    }

    public void setBatterySize(double b) {
        batterySize = b;
    }

    public double getBatterySize() {
        return batterySize;
    }

    public String toString() {
        return super.toString() + "Battery size: " + batterySize;
    }
}

